frontend/src/queries/nodes/DataNode/dataNodeLogic.ts

```diff
@@ -183,6 +194,13 @@ export const dataNodeLogic = kea<dataNodeLogicType>([
                 loadNextData: () => null,
             },
         ],
+        acknowledgedRefreshButtonChanged: [
+            false,
+            { persist: true },
```

Comment:
Ah, just noticed one problem here, which also applies to `insightLogic`: This logic is keyed per insight, so the acknowledgement state is also per insight. This would work as expected if we put the reducer on `insightSceneLogic` instead. I _think_ that should be alright, but if it's painful, then I guess back to `localStorage`

Reply:
There's this: https://github.com/keajs/kea-localstorage#storagekey